### PR TITLE
[16.0][CHG] fs_base_multi_image: drag and drop fixes

### PR DESCRIPTION
**Pull request by benwillig**
- create in memory records instead of calling the ORM explicitly

### CODE REVIEW

**Comment by lmignon** [comment]
@benwillig Why do we need to create the relation on drag and drop? Can we add a relation and save it when the parent is saved?

**Comment by benwillig** [comment]
@lmignon That's what I wanted to do, unfortunately I didn't find the right way to do it with the new framework.

**Comment by benwillig** [comment]
@lmignon I finally found the way to make it work as it was in the old widget. And it also works when saving a specific image directly on the relation record.

**Review by lmignon — APPROVED** [review]
Great! Works like a charm.

**Comment by lmignon** [comment]
@bealdav Can you take a look at this?

**Comment by bealdav** [comment]
Yes I 'll dive in.

**Comment by bealdav** [comment]
Thanks for this improvement even if don't understand the full details.

Nice to see Kanban view completed with some fields.

As product.product view doesn't allow to edit images, don't you think that this view could show kanban view instead of tree ? It's a better visual view for users.

**Comment by benwillig** [comment]
Yes we can display the kanban view by default. What do you think @lmignon ?

**Comment by lmignon** [comment]
> Yes we can display the kanban view by default. What do you think @lmignon ?

Yes we can on the product.product view.

**Review by bealdav — APPROVED** [review]
Thanks a lot. LGTM
Kanban in product.product view is a  nice to have  I think

**Comment by bealdav** [comment]
It works nicely in prod db, really thanks

**Comment by bealdav** [comment]
Also this PR https://github.com/OCA/storage/pull/370 seems ok. What do you think ?

File: fs_product_multi_image/views/product_template.xml [hunk 1]
```diff
@@ -20,17 +20,20 @@
...
+                        mode="kanban"
+                    >
                         <tree>
                             <field name="sequence" widget="handle" />
                             <field name="name" />
+                            <field name="link_existing" invisible="1" />
+                            <field name="image_id" invisible="1" />
+                            <field name="image" invisible="1" />
+                            <field name="specific_image" invisible="1" />
                         </tree>
```
Comment (lmignon) [inline fs_product_multi_image/views/product_template.xml:35]: Is-it the right way to define the kanban?

Comment (benwillig) [reply fs_product_multi_image/views/product_template.xml:35]: I added a specific kanban view for the image relation model so we don't need to define an inline kanban view here (if this is what you asked).

**Pull request merged**